packages/javascript-react/src/tests/setup-mock-api.ts

```diff
@@ -0,0 +1,104 @@
+import { Record } from "immutable";
+import { delay, HttpResponse, http } from "msw";
+import { setupServer } from "msw/node";
+
+interface SetupMockAPIOptions {
+    baseEndpoint: string;
+    resourceEndpoint?: string;
+    nestedBaseEndpoint?: string;
+}
+
+export function setupMockAPI({
```

Comment:
Should this be `setupMockApi` for naming case consistency? Same for `SetupMockApiOptions`.

Reply:
I'm cool with that